cat_emails/gmail_fetcher.py

```
@@ -1,17 +1,18 @@
+import openai
+import argparse
 from email import message_from_bytes
 from datetime import datetime, timedelta, timezone
 from email.utils import parsedate_to_datetime, parseaddr
 from typing import List, Optional, Set
 from bs4 import BeautifulSoup
-import re
 from collections import Counter
 from tabulate import tabulate
-from domain_service import DomainService, AllowedDomain, BlockedDomain, BlockedCategory
+from .domain_service import DomainService, AllowedDomain, BlockedDomain, BlockedCategory
```

Ruff (0.8.0)

`.domain_service.AllowedDomain` imported but unused

Remove unused import

(F401)